projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -133,12 +133,12 @@ public function admin_page() {
 						),
 					)
 				),
-				esc_url( $this->publicize->publicize_connections_url() )
+				esc_url( $this->publicize->publicize_connections_url( 'jetpack-social-connections-classic-editor' ) )
```

Comment:
I think this needs to be limited to classic view + proxy users?

@ebinnion Should we be sending non classic view atomic users to Jetpack cloud once we release this? (release = remove proxy requirement, but not the classic view requirement)

Reply:
> I think this needs to be limited to classic view + proxy users?

I believe it is due to code-D138559

Reply:
> We need to test this works to make a connection in calypso, toggle to classic view, see the same connection on jetpack cloud. If that doesn't work we'll need to implement connections in site tools.

You should see all the same connections whether you're in Calypso or Cloud, it's the same components and API calls.